### PR TITLE
Clean up TextareaField

### PR DESCRIPTION
**Pull request by assertchris**

### CODE REVIEW

File: forms/TextareaField.php [hunk 1]
```diff
@@ -14,58 +13,96 @@
...
 	public function setColumns($cols) {
 		$this->cols = $cols;
+
 		return $this;
 	}
 
+	/**
+	 * @return int
+	 */
+	public function getColumns() {
```
Comment (tractorcow) [inline forms/TextareaField.php]: If it's a new public API method it probably needs to go into 3.2 (3 branch).

**Comment by tractorcow** [comment]
See my comment on https://github.com/silverstripe/silverstripe-framework/pull/4117 about duplicate PHPDoc in subclasses.

File: forms/TextareaField.php (hunk 1 above)
Comment (assertchris) [inline forms/TextareaField.php]: ARG. Do you want me to remove these getters for this PR?

Comment (tractorcow) [inline forms/TextareaField.php]: I don't _want_ you to remove it.

I simply require you to. :D

**Comment by assertchris** [comment]
@tractorcow ping

**Pull request merged**